### DISCUSSION TITLE
Hacktoberfest Intiative Ideas

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#187] [Monthly Challenges]
@Virtual-Coffee/monthly-challenge @Virtual-Coffee/lightning-talk-team @Virtual-Coffee/ideas 

Hey friends! Hacktoberfest will be here before we know it, and we're really excited about it!

Last year, @danieltott gave us a brand new site and made the issue for our members to add themselves to the site.

We'd love to brainstorm ideas for a new issue or issues that members could take part in or any other Hacktoberfest ideas.

It's important to us that we support developers at all levels, but we also plan on having mentors for (hopefully) everyone who requests a mentor.

So I'm pulling you all in to help us get the idea train started. I'm up for any and all ideas!

**Comment by tkshill** [comment]
Roger had an idea concerning bots and mentorship, and is willing to take point on it. I want to get him in a meeting with some contributors at some point.

**Comment by calendee** [comment]
How about finding a charity and build something useful for them?

- Public website
- Internal admin tool
- ???

For example, right now, I'm building a tool to help a local community kitchen take inventory using a barcode scanner.

It would be awesome if we could support a charity that a VCer is already involved in.

**Reply by BekahHW** [reply]
We're looking into this for the Hackathon in early 2022. For the hackathon, we need to try to make this as accessible to different backgrounds as possible.

Last year, the first PR was adding your name to the site. And then we had at least four members, plus Forem and NickyT, have open issues on their repos. We do not have to do the same thing. But we very much viewed it as a learning experience and tried new languages, new roles, and learned together. 

My worry in focusing on one project with a solid goal is that people will feel more pressure and be less likely to contribute--if that makes sense.

**Reply by clandau** [reply]
I think last year went really well

**Comment by ClJarvis** [comment]
Is there any thing that VC needs built? What could help VC run better? anything that could be automated? 

How about a showcase of VC member's portfolios? We have web site links on our member bios. But maybe add a page that has a preview link to someone's port. Then rotate members every month.

**Comment by BekahHW** [comment]
We have talked about having a place for members who livestream, and members
who create content too. I wonder if we can tie this together.

**Comment by tkshill** [comment]
I feel like I want to go slow to go fast.

- Add your name to the site was great last year and I think we have a whole new crop of members to go through that process. That issue feels perennial.
- We had forms last time for sourcing Open source projects in the community and it would good to use those again, as well as see who from last year are still interested in participating. 
- These alone put us on par with last year, but now with more participants/support. And what I'm wondering is what features from last year can be made even better? 
- Can we improve the contributing docs?
- Or maybe incorporate video guides like loom more?
- Is there a better way to sync mentors with new contributors?

There's lots of cool things we could do with VC, but I think we have to be super careful which ones we wrap into Hacktoberfest. It may be that those become ongoing projects that start running _after_ we generate support from Hacktoberfest.

**Reply by BekahHW** [reply]
@tkshill I like the idea of having looms for different projects. If we had a central page on the site maybe for Hacktoberfest that gave a link to the repo and/or included a loom that could make it easier to access projects.

- It would be awesome to have a volunteer coordinator team who pairs mentors and contributors. Or we have a list of mentors first and allow contributors who want mentors to choose themselves. So we'd need more of a signup form for that. But I'd still like a coordinator to check in with pairs to ensure everyone is doing ok.
- Potentially, we could make @ClJarvis project for the VC Guide part of Hacktoberfest--Oh! Maybe that's the VC side of things. We can create the members' resources section of the site (tagging @danieltott for any thoughts)

**Reply by ClJarvis** [reply]
I thought the VC Guide might be a good place for some HacktoberFest PRs. Do we need most of it up before then so that it can help newer members. I expect an influx of new members during that time. Should we select parts to do now and save somethings for Hacktober?
While brainstorming the VC guide an FAQs was mentioned. If that was done now would it help the maintainers come October?

**Comment by aurelieverrot** [comment]
It seems things are getting warmer and people start to get excited by the upcoming HacktoberFest.

I have read all the messages I had everywhere (I think) and I have made a summary to keep the discussion moving:
(let me know if there are things missing or already done that I missed)

### "Add yourself as a member" was a huge success last year. And it seems we want to keep it this year.
- From last year's experience, do we need to improve something on this issue? Do we keep the [same issue as last year](https://github.com/Virtual-Coffee/virtualcoffee.io/issues/13) or open a new one? 
- Do we need to add any labels to the issue to make sure it counts for the Fest?
- Is there a need for support on the maintainer side to approve the PRs?
- Who is willing to host a workshop for it? I think we need one person to present, one or two to monitor the chat and answer questions, and one or two backup helpers.
- And we need a date for this workshop

### Mentoring / Help & Pairing
- How and when to provide a list of mentors and mentees?
- How to connect people?
- Bekah mentioned a special HacktoberFest co-working room earlier

### List of repos
- Did someone get any feedback on this? Or ideas?
- Do we want to have a VC-backed list? (list of repos from members that we know answer all the items we listed in the maintainer checklist)

**Comment by aurelieverrot** [comment]


**Reply by aurelieverrot** [reply]
- Lucia shared StepZen repo. They are an active contributing company during HacktoberFest:
https://virtual-coffee-group.slack.com/archives/C01DYTKRYAY/p1631303197091800
Stack is GraphQL, React, Nuxt.JS (Vue)

**Reply by danieltott** [reply]
@aurelieverrot - @BekahHW started this discussion for people to add to: https://github.com/Virtual-Coffee/virtualcoffee.io/discussions/321

**Reply by aurelieverrot** [reply]
😭 I knew I saw something like that, couldn't remember where 😭

Thank you!
